FluentEmailSample/Program.cs

```diff
@@ -8,7 +8,29 @@
 
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddSwaggerGen();
+builder.Services.AddSwaggerGen(options =>
+{
+    options.SwaggerDoc("v1", new OpenApiInfo
+    {
+        Title = "Fluent Email",
+        Version = "v1",
+        Description = "A Web API that uses FluentEmail to send e-mails, with the support for Sendinblue",
+
+        Contact = new OpenApiContact
+        {
+            Name = "Marco Minerva"
+        },
+
+        License = new OpenApiLicense
+        {
+            Name = "Licenza MIT",
+            Url = new Uri("https://it.wikipedia.org/wiki/Licenza_MIT"),
+        }
+    });
+
+    options.UseAllOfToExtendReferenceSchemas();
+    options.IncludeXmlComments(Path.Combine(AppContext.BaseDirectory, $"{Assembly.GetExecutingAssembly().GetName().Name}.xml"));
```

Comment:
What is the benefit of including XML comments without actually writing XML documentation in the source code?